modules/efk-logging-fluentd-alerts.adoc

```diff
@@ -2,44 +2,57 @@
 //
 // * logging/efk-logging-fluentd.adoc
 
-[id="efk-logging-fluentd-log-viewing_{context}"]
-= Viewing Fluentd logs
-
-How you view logs depends upon the `LOGGING_FILE_PATH` setting.
-
-* If `LOGGING_FILE_PATH` points to a file, the default, use the *logs* utility, from the project, 
-where the pod is located, to print out the contents of Fluentd log files:
-+
-----
-$ oc exec <any-fluentd-pod> -- logs <1>
-----
-<1> Specify the name of a Fluentd pod. Note the space before `logs`.
-+
-For example:
-+
-----
-$ oc exec fluentd-ht42r -n openshift-logging -- logs
-----
-+
-To view the current setting:
-+
-----
-oc -n openshift-logging set env daemonset/fluentd --list | grep LOGGING_FILE_PATH
-----
-
-* If you are using `LOGGING_FILE_PATH=console`,  Fluentd writes logs to stdout/stderr`. 
-You can retrieve the logs with the `oc logs [-f] <pod_name>` command, where the `-f`
-is optional, from the project where the pod is located.
-+
-----
-$ oc logs -f <any-fluentd-pod> <1>
-----
-<1> Specify the name of a Fluentd pod. Use the `-f` option to follow what is being written into the logs.
-+
-For example
-+
-----
-$ oc logs -f fluentd-ht42r -n openshift-logging
-----
-+
-The contents of log files are printed out, starting with the oldest log. 
+[id="efk-logging-fluentd-alerts_{context}"]
+= About logging collector alerts
+
+The following alerts are generated by the logging collector and can be viewed on the *Alerts* tab of the Prometheus UI.
+
+If an alert is triggered, it appears on the *Monitoring* -> *Alerts* page of the {product-title} web console.
+
+.Fluentd Prometheus alerts
+|===
+|Alert |Description |Example
+
+|`fluentdHealthRED`
+|Prometheus cannot scrape the Fluentd instance for more than the configured number of minutes, default 10.
+|`Fluentd cannot be scraped`
+
+|`fluentdIncreasedQLength`
+|The Fluentd buffer queue length increased more than the configured length in 1 minute,
+`raise warning alert`
+|`Fluentd is overwhelmed`
+
+|`fluentdQGrowingTime`
+|The Fluentd buffer queue length constantly increased for more than the specified time, default 12 hours.
+|`Fluentd file buffer usage issue`
+
+|`fluentdNumErrors`
+|Fluentd issues more than the specified number of errors, default 10.
+|`Fluentd reports high number of errors`
+
+|===
+
+.Rsyslog Prometheus alerts
+|===
+|Alert |Description |Example
```

Comment:
Alert rules for rsyslog: https://github.com/openshift/cluster-logging-operator/blob/master/files/rsyslog/rsyslog_prometheus_alerts.yaml